ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -282,6 +282,7 @@ protocolInfoCardano
   -> Update.ProtocolVersion
   -> Update.SoftwareVersion
   -> Maybe PBftLeaderCredentials
+  -> History.SafeBeforeEpoch
```

Comment:
Do the `Maybe EpochNo` change here too, but the big comment is not needed (internal module).

Reply:
Done!